integration_tests/integration/excludeDates.cy.ts

```diff
@@ -0,0 +1,35 @@
+import HomePage from '../pages/home'
+import Page from '../pages/page'
+import BlockedVisitPage from '../pages/blockedVisits'
+import TestData from '../../server/routes/testutils/testData'
+
+context('Exclude dates', () => {
```

Comment:
```suggestion
context('Block visit dates', () => {
```

Unfortunately we've got duplicate names (exclude/block) for this feature given API endpoint names. The story and UI features use "block" so my suggestion is we use this in the UI code as far as possible.

So, changing use of 'exclude' here and renaming file to `blockVisitDates.cy.ts`